it/uk/gov/hmrc/incorporatedentityidentificationfrontend/controllers/CheckYourAnswersControllerISpec.scala

```diff
@@ -476,6 +480,10 @@ class CheckYourAnswersControllerISpec extends ComponentSpecHelper
         stubAuth(OK, successfulAuthResponse(Some(testInternalId)))
         stubAudit()
         stubRetrieveCompanyProfileFromBE(testJourneyId)(status = OK, body = Json.toJsObject(testCompanyProfile))
+        stubValidateIncorporatedEntityDetails(testCompanyNumber, None)(BAD_REQUEST, Json.obj(
+          "code" -> "NOT_FOUND",
+          "reason" -> "The back end has indicated that CT UTR cannot be returned")
+        )
         stubStoreIdentifiersMatch(testJourneyId, identifiersMatch = false)(status = OK)
         stubStoreRegistrationStatus(testJourneyId, RegistrationNotCalled)(status = OK)
         stubRetrieveBusinessVerificationStatus(testJourneyId)(status = OK, body = testBusinessVerificationJson(value = businessVerificationNotEnoughInfoToChallengeKey))
```

Comment:
Would this BV status be NOT_FOUND as BV is disabled? Also verifyStoreBusinessVerificationStatus here